Sources/CreateAPI/Generator/Generator+DataTypes.swift

```diff
@@ -7,6 +7,8 @@ extension Generator {
         switch format {
         case .byte:
             return builtInType("Data", format: "byte", overrides: options.dataTypes.string)
+        case .binary:
+            return builtInType("Data", format: "binary", overrides: options.dataTypes.string)
```

Comment:
I think this is fine. It's a breaking change but it's minor and there is a way to revert to previous behaviour. I'll add a note in the release notes:

> `string` properties in a schema with the `format` set to `binary` will now be generated as `Data` types instead of `String`. To revert back to the previous behaviour, use the [`dataTypes`](https://github.com/CreateAPI/CreateAPI/blob/main/Docs/ConfigOptions.md#datatypes) configuration option:
>
> ```yml
> dataTypes:
>   string:
>     binary: String
> ```